control-plane-operator/controllers/hostedcontrolplane/cvo/reconcile.go

```diff
@@ -68,6 +68,8 @@ var (
 		"0000_50_cluster-ingress-operator_02-deployment-ibm-cloud-managed.yaml",
 		"0000_70_cluster-network-operator_02_rbac.yaml",
 		"0000_70_cluster-network-operator_03_deployment-ibm-cloud-managed.yaml",
+		"0000_80_machine-config-operator_01_machineconfig.crd.yaml",
```

Comment:
oh I thought we were excluding this already, do we have a counterpart PR to exclude them properly via profile?

Reply:
we don't yet, but I didn't want to make that change without making sure that it won't negatively affect ROKS clusters which use the same profile